### DISCUSSION TITLE
May I ask what is the difference in the first parameter of the API for setting the projection matrix between the two cameras below? When should the first API be used and when should the second API be used

### DISCUSSION THREAD

**Discussion by Haidiye00** [google/filament#7016] [Q&A]
![BJidjQEte6](https://github.com/google/filament/assets/132528822/50e9f066-f3d8-47e4-86e1-91760628ab8a)
I mostly use the API of first parameters is fov

**Comment by romainguy** [comment]
It's explained in the documentation. But in short those two APIs do the same thing except one lets you define the projection on photographic terms (i.e. focal length instead of field of view angle).
